### PR TITLE
package.json added so it can be installed using npm

### PR DESCRIPTION
**Pull request by namila**
package.json added so it can be installed using npm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "dependencies":{
+    "jquery": ">=1.6"  
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/namila/jQuery-Validation-Engine.git"
```
Comment (dennybrandes) [inline package.json]: Url is wrong.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/namila/jQuery-Validation-Engine/issues"
```
Comment (dennybrandes) [inline package.json]: Should point to original repo not to you own.

**Review by dennybrandes — DISMISSED** [review]
Please  check this file. It is based on your repository, not on the original one.

**Review by dennybrandes — CHANGES_REQUESTED** [review]
Please check what was mentioned in comments

File: package.json (hunk 1 above)
Comment (namila) [reply package.json]: corrected :)

File: package.json (hunk 2 above)
Comment (namila) [reply package.json]: corrected :)

**Comment by namila** [comment]
I updated the pull request with correct URL

**Comment by jacquesg** [comment]
Can this be merged and uploaded to npm?

**Comment by namila** [comment]
Thanks a lot :)

**Comment by jacquesg** [comment]
Thank you for merging this! Could you upload to npm?

**Comment by wblondel** [comment]
Could this be uploaded to npm?

**Comment by cokeito** [comment]
is this available via npm?

**Pull request merged**